### DISCUSSION TITLE
mac的这个界面如何调整显示的字体大小

### DISCUSSION THREAD

**Discussion by HolyDamn9527** [windingwind/zotero-better-notes#1221] [Q&A]
插件在mac的整体显示偏小，比zotero的界面字体还要小，是否可以调整显示比例或者字体

**Comment by AnitaLeungxx** [comment]
Windows同问。。字太小了。。

**Reply by AnitaLeungxx** [reply]
刚刚好像找到了能解决这个问题的方法。如果你是觉得笔记字号小的话可以参考：“设置”-“高级”-“设置编辑器”-在搜索框搜索fontsize然后再写着“note”那一栏修改字号就好
![image](https://github.com/user-attachments/assets/7cb37e5e-39ad-4076-b9ac-57f64a4a9aad)
![image](https://github.com/user-attachments/assets/67feb438-f7f3-4d7b-8080-ad4aa840bb90)

**Reply by windingwind** [reply]
用不着这么麻烦，顶部菜单栏-视图-笔记字体大小直接可以调整的

<img width="531" alt="image" src="https://github.com/user-attachments/assets/dab51d11-00af-4d30-b8af-7d7ce4c0f5e7" />

**Comment by windingwind** [comment]
具体是什么地方字小，请截图
